core/blockly.js

```diff
@@ -121,6 +121,12 @@ Blockly.dragMode_ = Blockly.DRAG_NONE;
 Blockly.onTouchUpWrapper_ = null;
 
 /**
+ * Which touch events are we currently paying attention to?
+ * @type {?DOMString}
```

Comment:
Objects are nullable by default (unlike primitives), so ? is not needed.
I think we need a @private tag here though.